src/content/blog/2024-10-17-the-data-process-that-powers-elections-in-utah.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: The Data Process That Powers Elections in Utah
+author: Miriam Seely
+description: >-
+  As you prepare for election day this year, learn more about how geospatial
+  data impacts where, when, and how you vote in Utah.
+date: '2024-10-17T16:31:16.697Z'
+category: UGRC
+cover_image: /src/images/pillar-blog/default-social-card.png
+cover_image_alt: ugrc social card
+---
+
+import Contacts from '@components/page/Contacts.astro';
+
+With the cool winds blowing in and the birds flying south, fall is officially upon us. That means a few things: warm sweaters, yellow aspen leaves, and everyone's favorite method of getting a free sticker: voting! Election Day will be on Tuesday, November 5th, 2024. If you aren't yet registered, do it now on [vote.utah.gov](http://vote.utah.gov) and vote by mail or in person! Did you know that where you vote and who's on your ballot are determined by geospatial data maintained by UGRC and local government authorities?
+
+UGRC might not be able to make the beautiful autumn hues of red, orange, and yellow stay longer than their natural time, but we are charged with safekeeping the data that makes elections in Utah possible. An example of these data are the [digital polygon representations](src\pages\products\sgid\political\voter-precincts.astro) of voting precinct boundaries. Maintaining these boundaries requires collaboration, organization, and a great attention to detail. Part of [our mission](/about/) as geospatial leaders in Utah is to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy, does making elections happen demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.
```

Comment:
```suggestion
UGRC might not be able to make the beautiful autumn hues of red, orange, and yellow stay longer than their natural time, but we are charged with safekeeping the data that makes elections in Utah possible. An example of these data are the [digital polygon representations](/products/sgid/political/voter-precincts/) of voting precinct boundaries. Maintaining these boundaries requires collaboration, organization, and a great attention to detail. Part of [our mission](/about/) as geospatial leaders in Utah is to “ensure a high level of coordination among Utah GIS users and effective, efficient use of GIS resources.” And boy oh boy, does making elections happen demand a lot of coordination. So, let's take a moment and get to know what's involved with keeping the data side of democracy alive.
```

relative links are URL links and not file system links. You don't need src\pages and astro extensions.